components/FrontPage/topbar.jsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import React from "react";
+import Btn from "./button";
+import ThemeSwitcher from "./themeSwitcher";
+import { useState, useEffect } from "react";
+
+export default function TopBar() {
+  // Logic for changing the scrollbar whenever it's not at the very top
+  const [isAtTop, setIsAtTop] = useState(true);
+  useEffect(() => {
+    const handleScroll = () => {
+      const scrollTop =
+        window.pageYOffset || document.documentElement.scrollTop;
+      setIsAtTop(scrollTop === 0);
+    };
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
```

Comment:
We should replace this with a `useScrollPosition` hook.

https://designcode.io/react-hooks-handbook-usescrollposition-hook